safe/gis/expressions.py

```diff
@@ -39,7 +39,7 @@ def inasafe_impact_analysis_layer(field, feature, parent):
 
     index = layer.fieldNameIndex(field)
     if index < 0:
-        return None
+        return index
```

Comment:
Yes you are right, I tried:
```
def inasafe_place_value_coefficient(number, feature, parent):
    """Given a number, it will return the coefficient of the place value name.

    For instance:
    *  inasafe_place_value_coefficient(10) -> 1
    *  inasafe_place_value_coefficient(1700) -> 1.7

    It needs to be used with inasafe_number_denomination_unit.
    """
    _ = feature, parent  # NOQA
    print type(number)
    print number
```

`inasafe_place_value_coefficient(None)` is not evaluated by QGIS, it doesn't print anything. But it works well with anything else. So can we tweak the expression in the expression builder?